nvm.sh

```diff
@@ -1590,10 +1591,15 @@ nvm_install_binary() {
   if [ -f "${TARBALL}" ]; then
     TMPDIR="$(dirname "${TARBALL}")/files"
   fi
+  local tar
+  tar='tar'
+  if [ "${NVM_OS}" = 'aix' ]; then
+    tar='gtar'
+  fi
   if (
     [ -n "${TMPDIR-}" ] && \
     command mkdir -p "${TMPDIR}" && \
-    command tar -x${tar_compression_flag}f "${TARBALL}" -C "${TMPDIR}" --strip-components 1 && \
+    command $tar -x${tar_compression_flag}f "${TARBALL}" -C "${TMPDIR}" --strip-components 1 && \
```

Comment:
please surround the tar variable in quotes, and use curly braces for the expansion